fingerprint/infra/nec-bio-sdk/src/main/java/com/simprints/fingerprint/infra/necsdkimpl/NecSdkModule.kt

```diff
@@ -0,0 +1,57 @@
+package com.simprints.fingerprint.infra.necsdkimpl
+
+import com.simprints.fingerprint.infra.basebiosdk.FingerprintBioSdk
+import com.simprints.fingerprint.infra.necsdkimpl.acquisition.image.FingerprintImageProviderImpl
+import com.simprints.fingerprint.infra.necsdkimpl.acquisition.template.FingerprintTemplateAcquisitionSettings
+import com.simprints.fingerprint.infra.necsdkimpl.acquisition.template.FingerprintTemplateMetadata
+import com.simprints.fingerprint.infra.necsdkimpl.acquisition.template.FingerprintTemplateProviderImpl
+import com.simprints.fingerprint.infra.necsdkimpl.initialization.SdkInitializerImpl
+import com.simprints.fingerprint.infra.necsdkimpl.matching.FingerprintMatcherImpl
+import com.simprints.fingerprint.infra.necsdkimpl.matching.NecMatchingSettings
+import com.simprints.necwrapper.nec.NEC
+import com.simprints.sgimagecorrection.SecugenImageCorrection
+import com.ygoular.bitmapconverter.BitmapConverter
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import javax.inject.Qualifier
+import javax.inject.Singleton
+
+@Qualifier
+@Retention(AnnotationRetention.BINARY)
+annotation class NecSdk
+
+@Module
+@InstallIn(SingletonComponent::class)
+object NecSdkModule {
+
+    @Provides
+    @Singleton
+    fun provideFingerprintBioSdk(
+        sdkInitializer: SdkInitializerImpl,
```

Comment:
IMO, all the implementation files should be internal by default, so this might not work.